src/app/pages/charging-stations/charging-limit/static-limit/charging-station-static-limit.component.html

```diff
@@ -11,7 +11,7 @@
   </div>
   <div class="row">
     <div class="col-md-6" *ngFor="let chargePoint of chargingStation?.chargePoints">
-      <div class="card" *ngIf="!chargePoint?.excludeFromPowerLimitation">
+      <div class="card" *ngIf="hasChargePointStaticLimitationSupport(chargePoint)">
```

Comment:
No method in 'if' statement in template, Angular will poll the method many times, use properties